### PR TITLE
Created angular js of playground

### PR DESCRIPTION
**Pull request by EmmanuelObua**

### CODE REVIEW

**Comment by twschiller** [comment]
Hi @EmmanuelObua, I see some changes for the other projects in this PR. This PR should only be impacting the angular js folder. Looking into the build issue now

File: source/angular-js/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+cd "$(dirname "${BASH_SOURCE[0]}")" # move to this script’s directory
+THIS_DIR=$(pwd)
+PROJECT_NAME="$(basename "$THIS_DIR")"
+PROJECT_DIST="$(cd ../../public && pwd)/$PROJECT_NAME"
+echo "Building '$PROJECT_NAME' into '$PROJECT_DIST'"
+### PLAYGROUND INTRO END
+
+cd repo
+npm install
+gulp build
```
Comment (fregante) [inline source/angular-js/build.sh:13]: This only works if gulp is installed locally. It's probably either one of this
```suggestion
npx gulp build
```
```suggestion
npm run build
```

**Comment by fregante** [comment]
@EmmanuelObua Don't run the `build.sh` file at the root:

https://github.com/pixiebrix/playground/blob/9454b09b4f5a5b0a4d623c1e111bf226247188bd/build.sh#L2

**Comment by EmmanuelObua** [comment]
> Hi @EmmanuelObua, I see some changes for the other projects in this PR. This PR should only be impacting the angular js folder. Looking into the build issue now

@fregante , i made changes to the git config for the changes in the directory of git submodules

**Comment by fregante** [comment]
What git config? The files from static should not *also* appear in the public folder on git. They're there because /build.sh was run locally and that shouldn't be necessary.

**Pull request closed without merging**